pkg/beholder/message.go

```diff
@@ -22,9 +23,9 @@ type Metadata struct {
 	// The version of the CL node.
 	NodeVersion string
 	// mTLS public key for the node operator. This is used as an identity key but with the added benefit of being able to provide signatures.
-	NodeCsaKey string
+	NodeCsaKey string `validate:"required"`
```

Comment:
@4of9, are these going to be added as resource attributes to beholder config or we want to pass them as regular attributes?

Reply:
I was thinkin `NodeCsaKey` would be included as a resource attribute so users don't have to add it to every message

Reply:
Same applies to `NodeCsaSignature`

Reply:
We might want to add validation for resource attributes as well. 
E.g check if `NodeCsaKey` is set then `NodeCsaSignature` should be set as well